backend/src/infra/database/database.manager.ts

```
@@ -64,12 +64,14 @@ export class DatabaseManager {
     try {
       const result = await client.query(
-        `SELECT column_name, data_type FROM information_schema.columns WHERE table_schema = $1 AND table_name = $2`,
+        `SELECT column_name, data_type, udt_name FROM information_schema.columns WHERE table_schema = $1 AND table_name = $2`,
         [schemaName, tableName]
       );
       const map: Record<string, string> = {};
       for (const row of result.rows) {
-        map[row.column_name] = row.data_type;
+        const dataType = row.data_type.toLowerCase();
+        map[row.column_name] =
+          dataType === 'user-defined' ? row.udt_name.toLowerCase() : dataType;
```

ESLint

[error] Delete `⏎·········`

(prettier/prettier)